src/utils/layout/NodesContext.tsx

```diff
@@ -27,7 +27,7 @@ interface NodesContext {
   setHiddenComponents: (mutator: (hidden: Set<string>) => Set<string>) => void;
 }
 
-function initialCreateStore() {
+export function initialCreateStore() {
```

Comment:
There's probably no need to export this one. I found no imports either, but now it indicates that it's being used somewhere else.